src/index.html

```diff
@@ -3,11 +3,46 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap"
+      rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main class="main">
+      <card class="card" data-qa="card">
+        <img class="card__image" src="./images/imac.jpeg"
+         alt="iMac 27">
+
+        <div class="card__title">
+          <h1 class="card__title-title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
```

Comment:
I think <h1> is little to strong here, please use h2

Reply:
if it is there only one h, it should be h1, I think ;) of course I know that it will be a part of catalog - so there it should be h2

Reply:
I totaly agree, it's better to change to h2 at least:
https://ahrefs.com/blog/h1-tag/

h1 is perfect for page title, but not for title of one small product card at page 😉